app/models/clickhouse/activity_log.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Clickhouse
+  class ActivityLog < BaseRecord
+    self.table_name = "activity_logs"
+    self.primary_key = nil
+
```

Comment:
Should we add the enum values here in this class?

```suggestion
      SOURCES = [
        :api,
        :front,
        :system
      ].freeze
      
      enum :activity_source, SOURCES
```

Reply:
Not sure we want this for clickhouse, but let's see